### ISSUE TITLE
Problems maintaining single fix

### ISSUE THREAD

**Issue by egorf** [tomojitakasu/RTKLIB#173]
Hi!

I have an issue with very unstable `single` fix. It comes and goes, dropping down to `-`. Rtkrcv error console says: `point pos error (chi-square error nv=20 vv=151.9 cs=34.5)`.

RTKLIB version if from master branch.

**Comment by tomojitakasu** [comment]
The chi-square validation failed indicates the quality of pseudorange is poor. At first, try in open-sky environment.

**Comment by egorf** [comment]
Yes, I your comment from issue #121. However this seems to happen in a good environment with antenna mounted on a rooftop. Is there anything I can do in order to get deeper into the problem?

**Comment by coandrei** [comment]
You may try to
1) check if any unhealthy satellite is used
2) use a ground plane for the patch antenna (if that’s what you use)
